sebs/statistics.py

```
@@ -6,31 +6,121 @@
+    # high_pos = ceil( 1 + (n + z * sqrt(n)) / 2 )
+    # These indices are 0-based for the sorted list.
+    
+    sqrt_n = math.sqrt(n)
+    if sqrt_n == 0: # Avoid division by zero if n=0, though caught by earlier check
+        return (np.nan, np.nan)
+
+    val_for_pos = z_critical_value * sqrt_n / 2.0
+    
+    # Ensure low_pos and high_pos are within valid array bounds [0, n-1]
+    # The formula can result in indices outside this range if n is too small.
+    low_idx = math.floor(n / 2.0 - val_for_pos)
```

Ruff (0.11.9)

Local variable `low_idx` is assigned to but never used

Remove assignment to unused variable `low_idx`

(F841)